Library/Paginate.swift

```diff
@@ -44,7 +44,6 @@ public func paginate<Cursor, Value: Equatable, Envelope, ErrorEnvelope, RequestP
   skipRepeats: Bool = true,
   valuesFromEnvelope: @escaping ((Envelope) -> [Value]),
   cursorFromEnvelope: @escaping ((Envelope) -> Cursor),
-  statsFromEnvelope: ((Envelope) -> Int)? = nil,
```

Comment:
Due to the very specific nature of needing stats here I've removed this and I am instead picking out this value where we need it.

Reply:
I think we added a ticket into the backlog to do this piece so you may have killed two birds with one stone on this PR.